Octokit.Tests.Integration/Helper.cs

```diff
@@ -228,5 +229,29 @@ public static IGitHubClient GetBadCredentialsClient()
                 Credentials = new Credentials(Guid.NewGuid().ToString(), "bad-password")
             };
         }
+
+        public static void DeleteInvitations(IConnection connection, List<string> invitees, int teamId)
+        {
+            try
+            {
+                foreach (var invitee in invitees)
+                {
+                    connection.Delete(new Uri($"orgs/{Organization}/memberships/{invitee}", UriKind.Relative), null, AcceptHeaders.OrganizationMembershipPreview).Wait(TimeSpan.FromSeconds(15));
```

Comment:
So I just tested it, and seems that `.Organization.Member.Delete` does *not* remove pending invitations